packages/atomic/src/components/search/atomic-text/atomic-text.styles.tw.css

```diff
@@ -0,0 +1,9 @@
+div {
+  transition: background-color 0.3s;
+
+  @apply text-white;
+}
+
+div:hover {
+  @apply bg-primary-dark;
+}
```

Comment:
These styles are just to demonstrate how to combine normal css, inline styles, tailwind @apply rules and tailwind classes directly in the component markup.